juspay_dashboard_mcp/tools.py

```diff
@@ -189,6 +189,13 @@
         handler=qapi.q_api,
         response_schema=None,
     ),
+    util.make_api_config(
+    name="rag_tool_juspay",
+    description="Query the RAG (Retrieval Augmented Generation) system using Vertex AI for intelligent document retrieval and question answering. Requires a 'query' parameter with the question to ask the RAG system. Optional 'similarity_top_k' parameter controls the number of similar documents to retrieve (default: 20).",
```

Comment:
can you give more info on what kind of RAG is this? mainly talk about the data/source for eg: juspay express checkout api docs.